ui/src/main/js/view/pipeline-staged.js

```diff
@@ -54,15 +54,25 @@ function _render(jobRunsData, onElement, fragCaption) {
     // The expected model is that produced by ui/src/main/js/model/runs-stage-grouped.js
 
     if (jobRunsData.runGroups && jobRunsData.runGroups.length > 0) {
+        var $ = jqProxy.getJQuery();
         var runGroup = jobRunsData.runGroups[0];
 
         runGroup.fragCaption = fragCaption;
         runGroup.maxTableEms = runGroup.stageData.length * 10;
 
         var pipelineStagedDom = templates.apply('pipeline-staged', runGroup);
-
         addLaneCharts(pipelineStagedDom, runGroup);
-        onElement.empty().append(pipelineStagedDom);
+        var viewPort = $('div.table-viewPort');
```

Comment:
Does that return the single element, or an array that always has a single element (as with the existing one)?  I'm not sure how smart the JQuery selectors are here.